spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/datastore/GcpDatastoreProperties.java

```diff
@@ -36,12 +36,25 @@
 	@NestedConfigurationProperty
 	private final Credentials credentials = new Credentials(GcpScope.DATASTORE.getUrl());
 
+	/** Configuration to auto configure a local Datastore Emulator. */
+	@NestedConfigurationProperty
+	private final EmulatorSettings emulator = new EmulatorSettings();
+
 	/**
-	 * The host and port of the local running emulator. If provided, this will setup the
-	 * client to connect against a running pub/sub emulator.
+	 * @deprecated use <code>spring.cloud.gcp.datastore.host</code> instead.
+	 * @see #host
 	 */
+	@Deprecated
 	private String emulatorHost;
 
+	/**
+	 * The host and port of the local running emulator as the following example:
+	 * localhost:8081. If provided, this will setup the client to connect against the
+	 * specified datastore server.
```

Comment:
You need to reconsider this long text to something much shorter.

All these comments on the configuration properties become as a content assistant in the IDE, so it's not so useful to let such a long text to pop up in the IDE.